dev-util/gitstrap/gitstrap-0.1.7.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Command line tool to bootstrap Github repository"
+HOMEPAGE="https://github.com/g4s8/${PN}"
+SRC_URI="${HOMEPAGE}/archive/${PV}.tar.gz -> ${PN}-${PV}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+DEPEND="dev-lang/go"
```

Comment:
It just hit me, isn't there some sort of eclass that makes packaging go packages a lot easier? Making whatever you do below redundant? (To be honest this ebuild looks a bit weird, but I've got no knowledge about go packages)